lib/src/components/Banner.tsx

```diff
@@ -9,13 +9,71 @@ type TBanner = {
 	image?: string,
 	primaryButton?: ReactElement,
 	secondaryButton?: ReactElement,
-	children: ReactElement | ReactElement[],
+	children?: ReactElement | ReactElement[],
 	canClose?: boolean,
 	onClose?: () => void,
 	onClick?: () => void,
-	maxHeight?: string,
```

Comment:
@MarcoWorms This was bad (my fault). I removed that to handle that with a specific react ref